docs/guide/migrating-from-vuepress.md

```diff
@@ -0,0 +1,105 @@
+# Migrating from VuePress
+
+VitePress follows VuePress config and default theme API where possible to ease the migration path. There are although some features that are not present in vitepress because of differences in the [design goals](../index.md) of the two projects. VitePress aims to have bare minimal features for authoring docs and most features are pushed to the Themes, where else VuePress has those features enabled by plugins.
+
+This is a list of changes and removed features compared to [VuePress v1.7.1](https://github.com/vuejs/vuepress/releases/tag/v1.7.1)
+
+::: warning
+
+Note this is early WIP! Currently the focus is on making Vite stable and feature complete first. It is not recommended to use this for anything serious yet.
+
+Some of these differences may be gone before Vitepress 1.0 is released.
+
+:::
+
+## General
+
+- YAML and TOML are not supported formats for site config. Only javascript is supported for `.vitepress/config.js`
+- removed [Plugins](https://vuepress.vuejs.org/plugin/) support, features are implemented in themes
+- removed [permalink support](https://vuepress.vuejs.org/guide/permalinks.html)
+- Components in `.vitepress/components` [are not auto registered as global components](https://vuepress.vuejs.org/)
+- [Public files](https://vuepress.vuejs.org/guide/assets.html#public-files) that are directly copied to dist root moved from `.vitepress/public/` to `public/`
+- Changed [App Level Enhancements](https://vuepress.vuejs.org/guide/basic-config.html#app-level-enhancements) API, app enhancements moved from `.vitepress/enhanceApp.js` to `.vitepress/theme/index.js`.
+- removed [styling](https://vuepress.vuejs.org/config/#styling) `.vitepress/styles/index.styl` and `.vitepress/styles/palette.styl`.
+- removed `.vitepress/templates`
+
+## Markdown
+
+- removed support for [toml in frontmatter](https://vuepress.vuejs.org/guide/frontmatter.html#alternative-frontmatter-formats)
+- removed [details block](https://vuepress.vuejs.org/guide/markdown.html#custom-containers)
+- removed [markdown slots](https://vuepress.vuejs.org/guide/markdown-slot.html)
+  guide/using-vue.html#using-components).
+- removed `~` prefix to explicitly specify a url is a [webpack module request](https://vuepress.vuejs.org/guide/assets.html#relative-urls)
+
+## Site Config
+
+- renamed `temp` to `tempDir`
```

Comment:
It might improve readability if you grouped the changes together, like:

- Renamed:
  - `temp` to `tempDir`
  - `dest` to `outDir`
...
- Removed:
  - `theme` from a dependency
  - `permalink`
  - `port`
...

Reply:
I agree, changed the PR to this structure. Thanks for the suggestions 💯